### PR TITLE
chore(ui): improve blocked TrackToggle

### LINKED ISSUE #1061: Improve blocked TrackToggle
As suggested in https://github.com/radicle-dev/radicle-upstream/pull/1047#issuecomment-710025267 a blocked TrackToggle should have a `not-allowed` cursor and show a tool tip explaining why the action can''t be performed.

### PR DESCRIPTION
**Pull request by xla**
Closes #1061

***
Misplaced Tooltip is addressed in #1070

### CODE REVIEW

File: ui/src/project.ts [hunk 1]
```diff
@@ -384,3 +384,9 @@ export const repositoryPathValidationStore = (
     );
   }
 };
+
+// Checks if the provided user is part of the maintainer list of the project.
+// FIXME(xla): Urns should be properly typed.
+export const isMaintainer = (project: Project, userUrn: string): boolean => {
+  return project.metadata.maintainers.includes(userUrn);
+};
```
Comment (NunoAlexandre) [inline ui/src/project.ts:392]:
```suggestion
export const isMaintainer = (userUrn: string, project: Project): boolean => {
  return project.metadata.maintainers.includes(userUrn);
};
```

To make the order of the arguments read better: `isMaintainer(userUrn, project)` <=> "is <user> maintainer of <project>"

File: ui/DesignSystem/Component/TrackToggle.svelte [hunk 2]
```diff
@@ -6,7 +6,7 @@
   import { Icon } from "../Primitive";
   import Hoverable from "./Hoverable.svelte";
 
-  export let disabled: boolean = false;
+  export let blocked: boolean = false;
```
Comment (NunoAlexandre) [inline ui/DesignSystem/Component/TrackToggle.svelte:9]: This change doesn't appear to add much in my view. We use "disabled" more often across the project so reverting this rename wouldn't be a bad idea.

**Review by NunoAlexandre — DISMISSED** [review]
Looks good. Left two suggestions for you to consider.

**Review by NunoAlexandre — DISMISSED** [review]

**Review by rudolfs — DISMISSED** [review]

**Review by cloudhead — APPROVED** [review]

**Review by rudolfs — APPROVED** [review]

**Review by FintanH — APPROVED** [review]
:metal:

**Pull request merged**